pkg/cluster/fetch.go

```diff
@@ -11,6 +11,30 @@ import (
 	v1alpha1release "github.com/aws/eks-anywhere/release/api/v1alpha1"
 )
 
+// ManagementComponents contain the versions bundle for a managment cluster's core components.
+type ManagementComponents struct {
+	*v1alpha1release.VersionsBundle
+}
```

Comment:
I would just copy over whatever are management components and have fields for it
This doesn't decouple upgraders from the format of a versions bundle

I mean something like

```suggestion
type ManagementComponents struct {
    Version string
	CoreCapi releasev1.Whatever
	DockerProvider releasev1.WhateverProvider
	...
	EKSAComponents releasev1.WhateverManifest
}
```